README.md

```diff
@@ -192,6 +194,19 @@ python prompt.py -e {experiment_name}
 where `experiment_name` is the output folder of the experiment you want to chat with (see configuration).
 The interactive chat will also work with model that were finetuned using the UI.
 
+To publish the model to Hugging Face, use the following command:
+
+```bash
+python publish_to_hugging_face.py -e {experiment_name} -d {device} -a {api_key} -u {user_id} -m {model_name}

```

Comment:
should require a `make shell` first, right?

Reply:
I'm not entirely certain. I gave it a try without using `make shell` and it seemed to work.

Reply:
It will only work when you installed huggingface hub etc. in your base environment

Reply:
Ok. Done.